.zombienet-tests/0001-custom.js

```diff
@@ -0,0 +1,31 @@
+const polkadotApi = require("@polkadot/api")
+
+async function connect(customChainSpec, types) {
+  const substrateConnect = await import("@substrate/connect")
+  const provider = new substrateConnect.ScProvider(customChainSpec)
+  const api = new polkadotApi.ApiPromise({ provider, types })
+  await api.isReady
+  return api
+}
+
+async function run(nodeName, networkInfo) {
+  const { userDefinedTypes } = networkInfo.nodesByName[nodeName]
+  const customChainSpec = require(networkInfo.chainSpecPath)
+
+  // TODO: forkId generate an error in smoldot-light
```

Comment:
Please tell me these kind of things

Reply:
https://github.com/paritytech/smoldot/issues/1955

Reply:
Smoldot v0.5.15 (published in ~30mn) fixes this.

Reply:
Thanks @tomaka! I will remove this from the test.